packages/inter-protocol/src/vpool-xyk-amm/multipoolMarketMaker.js

```diff
@@ -189,10 +234,14 @@ const start = async (zcf, privateArgs) => {
 
   // For now, this seat collects protocol fees. It needs to be connected to
   // something that will extract the fees.
-  const { zcfSeat: protocolSeat } = zcf.makeEmptySeatKit();
+  const { zcfSeat: protocolSeat } = provide(baggage, 'protocolSeat', () =>
+    harden(zcf.makeEmptySeatKit()),
+  );
 
   /** @type {AssetReservePublicFacet=} */
-  let reserveFacet;
+  let reserveFacet = baggage.has('reserveFacet')
+    ? baggage.get('reserveFacet')
+    : undefined;
```

Comment:
out of scope but if we do this a lot,
```suggestion
  let reserveFacet = maybeGet(baggage, 'reserveFacet'));
```
Or have something like that on `MapStore`. Though I suppose there were reasons to omit it.

Reply:
If we did this a lot, I'd be fine with a function like that, but I think the store API is this way because we're not expecting to do it a lot.